resources/elasticsearchservice-domain.go

```diff
@@ -28,6 +30,7 @@ func ListESDomains(sess *session.Session) ([]Resource, error) {
 		resources = append(resources, &ESDomain{
 			svc:        svc,
 			domainName: domain.DomainName,
+			tags: domain.Tags,
```

Comment:
```suggestion
			tags: domain.Tags,
```

Go is case-sensitive.